src/containers/pages/InterventionPlan/NewPlan/General/tests/index.test.tsx

```diff
@@ -52,9 +52,6 @@ describe('containers/pages/NewPlan', () => {
     wrapper
       .find('input[name="jurisdictions[0].name"]')
       .simulate('change', { target: { name: 'jurisdictions[0].name', value: 'Onyx' } });
-    wrapper
-      .find('select[name="interventionType"]')
-      .simulate('change', { target: { name: 'interventionType', value: 'FI' } });
```

Comment:
I believe this [line](https://github.com/onaio/reveal-frontend/blob/2241149c0ec9dae0aa746b168a02d05ec7e4107f/src/components/forms/PlanForm/index.tsx#L301) undoes the results of the below simulated changes.
```
    (wrapper
      .find('FieldInner')
      .first()
      .props() as any).formik.setFieldValue('jurisdictions[0].id', '1337');
    // set jurisdiction name
    wrapper
      .find('input[name="jurisdictions[0].name"]')
      .simulate('change', { target: { name: 'jurisdictions[0].name', value: 'Onyx' } });
```

 Removing this line does not have any consequences to the tests because `FI` is already the default value.